src/main/java/seedu/address/logic/commands/CreateGroupCommand.java

```diff
@@ -0,0 +1,66 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.List;
+import java.util.Optional;
+import java.util.logging.Logger;
+
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.person.Group;
+import seedu.address.model.person.Person;
+
+/**
+ * Creates a group in the address book.
+ */
+public class CreateGroupCommand extends Command {
+
+    public static final String COMMAND_WORD = "group";
+    public static final String MESSAGE_USAGE = COMMAND_WORD + " PREVIOUS_GROUP_NAME UPDATED_GROUP_NAME\n"
+            + "Example: " + COMMAND_WORD + " T01 T02";
+    public static final String MESSAGE_DUPLICATE_GROUP = "This group already exists in the address book";
+    public static final String MESSAGE_SUCCESS = "Group updated: %1$s";
+    private static final Logger logger = Logger.getLogger(CreateGroupCommand.class.getName());
+    private final String groupName;
```

Comment:
Perhaps you should have a message for missing PREVIOUS_GROUP?

Reply:
Thank you . I have updated the message as per requested.